SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialVirtualWorkerTranslator.cpp

```diff
@@ -92,6 +92,24 @@ void SpatialVirtualWorkerTranslator::AuthorityChanged(const Worker_AuthorityChan
 	}
 }
 
+// Check to see if this worker's physical worker name is in the mapping. If it isn't, it's possibly an old mapping.
```

Comment:
This approach is a bit of a hack to allow PiE restarts. It's not very efficient, but it should only be called < 10 times per worker on PiE restarts so I don't really care.

Reply:
Maybe worth adding your comment here as comments in code as it seems useful context?